docs/observability/metrics.md

```diff
@@ -117,6 +117,9 @@ Virtual Machine last transition timestamp to starting status. Type: Counter.
 ### kubevirt_vm_vnic_info
 Details of Virtual Machine (VM) vNIC interfaces, such as vNIC name, binding type, network name, and binding name for each vNIC defined in the VM's configuration. Type: Gauge.
 
+### kubevirt_vmi_cpu_pressure_seconds
```

Comment:
How is this different from the CPU delay metric?

Reply:
the cpu pressure provides way more details so we can better understand the types of delays that are happening as he have the `some` and `full` parts of the pressure stats

`some` gives us insights into the latency caused by the lack of a resource for one or more (some) tasks, this means the CPU was always busy

`full` means all tasks were waiting, so during these time reported by `full`, all tasks were waiting for some resource, which means unproductive time for the CPU